src/config/configManager.go

```diff
@@ -182,12 +182,14 @@ func LoadConfigFromFile() {
 	}
 
 	CurrentCfg.ConfigObservability = types.ConfigObservability{
-		Enable:              viper.GetBool("observability.enable"),
-		CronJobTimeInterval: "@every " + viper.GetString("observability.cron-job-time-interval"),
-		DBName:              viper.GetString("observability.dbname"),
-		SysObservability:    viper.GetBool("observability.system-observability"),
-		NetObservability:    viper.GetBool("observability.network-observability"),
-		WriteLogsToDB:       viper.GetBool("observability.write-logs-to-db"),
+		Enable:                viper.GetBool("observability.enable"),
+		CronJobTimeInterval:   "@every " + viper.GetString("observability.cron-job-time-interval"),
+		DBName:                viper.GetString("observability.dbname"),
+		SysObservability:      viper.GetBool("observability.system-observability"),
+		NetObservability:      viper.GetBool("observability.network-observability"),
+		WriteLogsToDB:         viper.GetBool("observability.write-logs-to-db"),
+		CronJobTimeForPurgeDB: "@every" + viper.GetString("observability.cron-job-time-for-purge-db"),
+		PurgeLogsFromDB:       viper.GetBool("observability.purge-logs-from-db"),
```

Comment:
```suggestion
		PurgeOldDBEntries:       viper.GetBool("observability.purge-logs-from-db"),
```

Reply:
Accepted and Thank you for suggesting and reviewing.